apps/web/src/lib/api/external/laskugeneraattori/actions.ts

```diff
@@ -0,0 +1,116 @@
+import {
+  type InvoiceGeneratorFormState,
+  type LaskugeneraattoriRequest,
+} from "./index";
+
+// https://stackoverflow.com/a/22015930
+const zip = <T>(...arr: T[][]) =>
+  Array(Math.max(...arr.map((a) => a.length)))
+    .fill(0)
+    .map((_, i) => arr.map((a) => a[i]));
+
+export const laskugeneraattoriBaseUrl =
+  process.env.NEXT_PUBLIC_LASKUGENERAATTORI_URL ??
```

Comment:
The env variable also needs to be added as a dependency in the root turbo.json

Reply:
Isn't it already there, or do I need to something else than add it to globalEnv? I did get some eslint errors from the env variable yesterday, but IDK what the problem is since the variable seems to be added in the same way as the ilmomasiina url variable for example 🤔

Reply:
Oh yep, the eslint issues are gone now so it was probably just some weird cache issue 🤔